### PR TITLE
Make getting started sample async

### PR DESCRIPTION
**Pull request by roji**

### CODE REVIEW

File: conceptual/Npgsql/index.md [hunk 1]
```diff
@@ -18,21 +18,21 @@ var connString = "Host=myserver;Username=mylogin;Password=mypass;Database=mydata
 
 using (var conn = new NpgsqlConnection(connString))
```
Comment (YohDeadfall) [inline conceptual/Npgsql/index.md]: Another `await` here maybe? (:

File: conceptual/Npgsql/index.md [hunk 2]
```diff
@@ -18,21 +18,21 @@ var connString = "Host=myserver;Username=mylogin;Password=mypass;Database=mydata
...
         cmd.Parameters.AddWithValue("p", "Hello world");
-        cmd.ExecuteNonQuery();
+        await cmd.ExecuteNonQueryAsync();
     }
 
     // Retrieve all rows
     using (var cmd = new NpgsqlCommand("SELECT some_field FROM data", conn))
-    using (var reader = cmd.ExecuteReader())
-        while (reader.Read())
+    using (var reader = await cmd.ExecuteReaderAsync())
```
Comment (YohDeadfall) [inline conceptual/Npgsql/index.md]: And two more here, what do you think?

File: conceptual/Npgsql/index.md (hunk 1 above)
Comment (roji) [reply conceptual/Npgsql/index.md]: Yeah, you're right (even if it doesn't make any difference for the common pooled usage).

**Comment by roji** [comment]
Also changed to use the new C# 8 using declarations.

It may be a bit too modern for most users, but they'll get used to it quickly enough :)

**Pull request merged**